rgbd/src/client_rgbd.cpp

```diff
@@ -32,6 +32,14 @@ bool ClientRGBD::intialize(const std::string& server_name)
 
 // ----------------------------------------------------------------------------------------
 
+bool ClientRGBD::deinitialize()
+{
+    sub_image_ = ros::Subscriber();
```

Comment:
Did you check if it actually unsubscribes? If it does (and stuff works as intended), please add a comment to emphasize that (lest anyone removes this statement for not understanding)

Reply:
Yes it unsubscribes. As the old subscriber's destructor is called. But also experienced it during tested.